controllers/kserve_inferenceservice_controller.go

```diff
@@ -468,6 +470,25 @@ func (r *OpenshiftInferenceServiceReconciler) createOrUpdateMetricsServiceMonito
 	return nil
 }
 
+func (r *OpenshiftInferenceServiceReconciler) DeleteKserveInferenceServiceRoute(ctx context.Context, inferenceService *kservev1beta1.InferenceService) error {
```

Comment:
shouldn't this be part of kisvc_route_reconciler?

Delete logic in two places is error prone.

Reply:
done... I had moved Route deletion logic to RouteHandler.